show_test.go

```diff
@@ -0,0 +1,119 @@
+package myradio
```

Comment:
Hmm. I'm more of a fan of black box testing with go modules tbh, so this should be myradio_test - can you fiddle it so that's possible?